XamlControlsGallery/ControlPages/AppBarPage.xaml.cs

```diff
@@ -42,8 +42,7 @@ private void NavBarButton_Click(object sender, RoutedEventArgs e)
         {
             ButtonBase b = (ButtonBase)sender;
 
-            Frame rootFrame = Window.Current.Content as Frame;
-            if (rootFrame != null && b.Tag != null)
+            if (Window.Current.Content is Frame rootFrame && b.Tag != null)
```

Comment:
Nit: Same here, `b` is not the best name and we are already changing the code here, so why not rename it too.